manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2

```

```suggestion
PackageVersion: 2.0.1.0
```
This should match what is written to control panel